catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/data/impl/ContentItemImpl.java

```diff
@@ -50,6 +53,8 @@
 
     private Metacard metacard;
 
+    protected String qualifier;
```

Comment:
To be sure I understand, this means that the content associated with a metacard consists of N `ContentItem`s? Not a single `ContentItem` with 0-N subcomponents?